README.md

```diff
@@ -28,24 +28,29 @@ For installation instructions head to the official documentation:
 
 ## Want to develop on Lektor?
 
-This gets you started (assuming you have Python, pip, Make and pre-commit
-installed):
-
-```
-$ git clone https://github.com/lektor/lektor
-$ cd lektor
-$ virtualenv venv
-$ . venv/bin/activate
-$ pip install --editable .
-$ make build-js
-$ pre-commit install
-$ export LEKTOR_DEV=1
-$ cp -r example example-project
-$ lektor --project example-project server
+This gets you started (assuming you have Python, pip, and Make installed):
+
+```shell
+git clone https://github.com/lektor/lektor
+cd lektor
+virtualenv venv
```

Comment:
While we're fixing these instructions, this should probably be updated to `python -m venv venv` (since the `venv` is in py3's  standard library, whereas `virtualenv` is an external dependency.)

----

Or, alternatively, we could use tox's `--devenv` option.  E.g., right now, if you do
```sh
tox --devenv venv
```
that will create a virtual environment with all the dependencies for the default [[testenv]](https://github.com/lektor/lektor/blob/949d9d8c04c82bdcb53f4a1b5a5a097aa63c4761/tox.ini#L26) installed (i.e. `pytest` and friends), and Lektor installed in editable mode, all in one go.

If we want to support installing the kitchen sink — more than what is required by any particularly testenv —we could include a "dev" testenv in the `tox.ini` (which doesn't actually do any testing)
```ini
[testenv:dev]
skip_install = true
commands =
deps =
    # pull in dependencies declared in other testenvs
    {[testenv]deps}
    {[testenv:lint]deps}
    # add some dependencies of our own
    pre-commit
```
so that
```sh
tox -e dev --devenv venv
```
will create a virtual environment with everything installed and ready to go.